src/DotNetEnv/Env.cs

```diff
@@ -7,6 +7,9 @@ namespace DotNetEnv
 {
     public class Env
     {
+
```

Comment:
No need for this initial newline whitespace

Reply:
Habit... removed